A/Aria2/build_tarballs.jl

```diff
@@ -50,11 +50,11 @@ dependencies = [
     # `LibSSH2_jll` was compiled with.  So we explicitly select the
     # right version here.
     BuildDependency(PackageSpec(name="MbedTLS_jll", version=v"2.28")),
-    Dependency(PackageSpec(name="OpenSSL_jll"); compat = "1.1.10"),
+    Dependency(PackageSpec(name="OpenSSL_jll"); compat="1.1.10"),
     Dependency(PackageSpec(name="XML2_jll")),
     Dependency(PackageSpec(name="Zlib_jll")),
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
-               julia_compat="1.6", preferred_gcc_version = v"7")
+    julia_compat="1.6", preferred_gcc_version=v"7")
```

Comment:
```suggestion
               julia_compat="1.8", preferred_gcc_version=v"7")
```

Reply:
Thank you, I missed the `clang_use_lld` option.